vm/chef/cookbooks/singlefs/files/retry_with_backoff.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+#
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Retry on error using exponential backoff.
+# 5 tries. Backoff: 10, 20, 40, 80 seconds.
+retry_with_backoff(){
```

Comment:
Is 2.5 mins timeout appropriate for any operation implemented with retry logic? It might make sense to have a parameter here for the timeout.

Reply:
PS. a function like this is a good candidate to be located in the common directory offered by c2d-config cookbook - https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/vm/chef/cookbooks/c2d-config.